server/handler.go

```diff
@@ -21,21 +29,35 @@ type Handler struct {
 	mu sync.Mutex
 	e  *sqle.Engine
 	sm *SessionManager
+	c  map[uint32]*mysql.Conn
```

Comment:
This is necessary? Adding a log line on `ComQuery` method, I can see the next behavior: